lib/router/src/utils.ts

```diff
@@ -9,7 +9,7 @@ export interface StoryData {
 
 const splitPathRegex = /\/([^/]+)\/(?:(.*)_)?([^/]+)?/;
 
-export const parsePath: (path?: string) => StoryData = memoize(1000)(
+export const parsePath: (path: string | undefined) => StoryData = memoize(1000)(
```

Comment:
I've made `path` not optional here, because  `memoize` throws `Memoizerific functions should always be called with the same number of arguments`